### DISCUSSION TITLE
Is there an equivalent to 'new_user_roles' for groups?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#4518] [Q&A] [answered]
Hi all!

Is there any equivalent to this setting in **config/statamic/users.php** 

```
    'new_user_roles' => [
        'roles' => 'member_pending'
    ],

```
for groups?

I want to give my client an easy shot at seeing all pending members in just one list. 

Which would be easy if I'd not only assign a role "member_pending" but an accompanying group 'members_pending' on registering – that way you could just view this group in CP clear of seeing already accepted or rejected members or superadmins (and than take appropriate actions on just these pending members one by one).

**Comment by jasonvarga** [comment, marked as answer]
Not at the moment. You could open an issue over at https://github.com/statamic/ideas

**Reply by jensolafkoch** [reply]
Thanks Jason!

How could I possible do the same without a config setting? Maybe there is a workaround?

**Reply by jasonvarga** [reply]
You could listen for the `UserSaved` event and apply the groups manually.

**Reply by jensolafkoch** [reply]
When I listen for `UserSaved` I can assign a new group to the user, but I have to save the new user which would emit another `UserSaved `event.

For a pure Laravel user I could use `saveQuietly()` instead of `save()` which is unfortunately not a method available to a Statamic User. (`https://laravel.com/docs/8.x/eloquent#muting-events` doesn't seem to work either.)

Cul-de-sac?

**Reply by jasonvarga** [reply]
Hang tight:
 - #4529
 - #3379

**Reply by jensolafkoch** [reply]
Great news, thanks for letting me (and all of us) know!

I guess that answers my question and I'm going to mark it accordingfly.
